### DISCUSSION TITLE
Add AWS as a deployment option in the documentation

### DISCUSSION THREAD

**Discussion by wirjo** [langfuse/langfuse#2863] [Ideas]
### Describe the feature or potential improvement

On: https://langfuse.com/docs/deployment/self-host#platform-specific-information

Add AWS as a deployment option in the documentation using AWS App Runner and AWS Aurora Serverless v2 due to low cost:
https://github.com/aaronsu11/langfuse-on-aws

**Comment by marcklingen** [comment]
Thank you for raising this. Do you think it makes sense to add this template in addition to the one that is already included in the docs?

https://langfuse.com/docs/deployment/self-host#aws
